app/models/pairing.rb

```diff
@@ -3,6 +3,18 @@ class Pairing < ApplicationRecord
   belongs_to :user2, class_name: 'User'
 
   validates :date_time, presence: true
-  validates :user1_cancelled?, presence: true
-  validates :user2_cancelled?, presence: true
+
+  def self.create_pairing(availability, open_slot)
+    pairing_time = determine_time_of_pairing(availability, open_slot[0])
+    Pairing.create!(user1: availability.user, user2: open_slot[0].user, date_time: pairing_time,
+                    user1_cancelled?: false, user2_cancelled?: false)
+  end
+
+  def self.determine_time_of_pairing(availability1, availability2)
+    if availability1.start_date_time > availability2.start_date_time
+      availability1.start_date_time
+    else
+      availability2.start_date_time
+    end
+  end
```

Comment:
This was the only way I could think of to schedule the time of the pairing. It will take the start time of whoever's available to start the latest

Reply:
Yeah I think that's fine for now